Source/WebKit/UIProcess/WebsiteData/Cocoa/WebsiteDataStoreCocoa.mm

```diff
@@ -907,7 +908,7 @@ static NavigatingToAppBoundDomain schemeOrDomainIsAppBound(const String& host, c
     if (path.isEmpty())
         path = NSHomeDirectory();
```

Comment:
It’s possible we can use `makeString` directly on the `NSString *` in this case for better efficiency.